.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy Miden Assembly playground
+
+on:
+  push:
+    branches:
+      - test-playground-deployment
```

Comment:
This should be either `main` or `next`, depending on when we want to deploy. It looks like we don't have a `next` branch yet, so maybe set it to `main`